### PR TITLE
refactor: shuttle-common/backend -> shuttle-backends

### PR DESCRIPTION
**Pull request by jonaro00**

### CODE REVIEW

File: common/src/limits.rs [hunk 1]
```diff
@@ -8,7 +8,7 @@ use crate::{
 #[derive(Clone, Debug, Deserialize, Serialize, Eq, PartialEq)]
 pub struct Limits {
```
Comment (chesedo) [inline common/src/limits.rs:9]: Only the backends need to know about this right?

File: deployer/src/handlers/mod.rs [hunk 2]
```diff
@@ -56,15 +54,16 @@ pub struct PaginationDetails {
 #[derive(Clone)]
 pub struct RouterBuilder {
     router: Router,
-    _project_name: ProjectName,
+    // might be used for tracing instruments?
+    _project_name: String,
```
Comment (chesedo) [inline deployer/src/handlers/mod.rs:58]: No idea. Should be better to delete

File: proto/Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "shuttle-proto"
-version.workspace = true
+version = "0.42.0"
```
Comment (chesedo) [inline proto/Cargo.toml:3]: Is this expected?

**Review by chesedo — COMMENTED** [review]
A few nits

File: proto/Cargo.toml (hunk 3 above)
Comment (jonaro00) [reply proto/Cargo.toml:3]: I made it follow the same pattern as the rest of the crates. No real world difference

File: deployer/src/handlers/mod.rs (hunk 2 above)
Comment (jonaro00) [reply deployer/src/handlers/mod.rs:58]: Don't want to risk breaking anything with this PR. No risk in keeping it.

**Review by iulianbarbu — APPROVED** [review]
LGTM!

File: deployer/src/handlers/mod.rs (hunk 2 above)
Comment (chesedo) [reply deployer/src/handlers/mod.rs:58]: Fair enough

File: proto/Cargo.toml (hunk 3 above)
Comment (chesedo) [reply proto/Cargo.toml:3]: Oky

File: common/src/limits.rs (hunk 1 above)
Comment (jonaro00) [reply common/src/limits.rs:9]: proto uses claims, not limits, but the limits file is part of the claims feature

File: common/src/models/error.rs [hunk 4]
```diff
@@ -6,6 +6,13 @@ use serde::{Deserialize, Serialize};
 use thiserror::Error;
 use tracing::{error, warn};
 
+#[cfg(feature = "axum")]
+impl axum::response::IntoResponse for ApiError {
+    fn into_response(self) -> axum::response::Response {
+        (self.status(), axum::Json(self)).into_response()
+    }
+}
```
[the remark this answers is not in the document]
Comment (jonaro00) [reply common/src/models/error.rs:14]: Yes, but the trait impl must exist in the same crate, and the model is shared with the "frontends". This is a simple compromise, but can be solved in other ways.

**Pull request merged**